content/sql/concepts/sql-injections/sql-injections.md

```diff
@@ -0,0 +1,41 @@
+---
+Title: 'SQL Injections'
+Description: 'SQL Injection is a technique to inject malicious code into a database.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Database'
+  - 'Queries'
+  - 'SQLite'
+  - 'MySQL'
+  - 'PostgreSQL'
+CatalogContent:
+  - 'learn-sql'
+  - 'paths/analyze-data-with-sql'
+  - 'paths/design-databases-with-postgresql'
```

Comment:
```suggestion
  - 'paths/analyze-data-with-sql'
```

Let's always use just one `learn-` link and one `paths/` link under `CatalogContent`, going with the most relevant subject matter.